### ISSUE TITLE
Problems in the step: "Add the Newtonsoft.Json NuGet package to the WeatherApp project".  

### ISSUE THREAD

**Issue by mearistizabal** [MicrosoftDocs/visualstudio-docs#649]
I am using Visual Studio 2017 Community and when I try to Add the Newtonsoft.Json Nuget, the shared code Weatherapp it's not available. I can only add it to the platform specific packages: Android, IOS. Any idea why is this happening and the solution?. Thank you.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3b25d4cd-0c3f-e9a6-6256-ea54d0972d4b
* Version Independent ID: babd49b7-8e18-13dc-99f2-6783495cb0dc
* [Content](https://docs.microsoft.com/en-us/visualstudio/cross-platform/learn-app-building-basics-with-xamarin-forms-in-visual-studio#feedback)
* [Content Source](https://github.com/Microsoft/visualstudio-docs/blob/master/docs/cross-platform/learn-app-building-basics-with-xamarin-forms-in-visual-studio.md)
* Service: unspecified

**Comment by gewarren** [comment]
@conceptdev can you answer this question?

**Comment by mearistizabal** [comment]
The same thing happens for Xamarin.forms.

**Comment by mearistizabal** [comment]
Hi. I started the process all over again and it worked this time, at least the NuGet process. It seems that the previous setup I didn't check the option ".NET standard" under "Sharing coding strategy".  It's the only difference I see. Thank you for your interest.

**Comment by conceptdev** [comment]
@mearistizabal Yes, we recommend using a **.NET Standard** project, which will let you reference NuGet packages.

For future reference, the difference with a **Shared Project** is that it does not compile into a separate assembly, instead all the C# code in a Shared Project is _included inside_ any referencing app projects during compilation. For this reason, you cannot reference a NuGet (or any other .NET assembly) from a Shared Project -- instead, the code will implicitly reference the NuGets and other assemblies referenced by app projects that _use_ the Shared Project.
